src/components/Section/Foreign/Activities/DirectActivity/DirectInterest.jsx

```diff
@@ -49,107 +29,109 @@ export default class DirectInterest extends ValidationElement {
       RelinquishedNotApplicable: this.props.RelinquishedNotApplicable,
       Explanation: this.props.Explanation,
       CoOwners: this.props.CoOwners,
-      ...queue
+      ...queue,
     })
   }
 
-  updateInterestTypes(values) {
-    let interestType = values.value
-    let selected = [...((this.props.InterestTypes || {}).values || [])]
+  updateInterestTypes = (values) => {
+    const interestType = values.value
+    const selected = [...((this.props.InterestTypes || {}).values || [])]
     if (selected.includes(interestType)) {
       selected.splice(selected.indexOf(interestType), 1)
     } else {
       selected.push(interestType)
     }
 
     this.update({
-      InterestTypes: { values: selected }
+      InterestTypes: { values: selected },
     })
   }
 
-  updateInterestType(values) {
+  updateInterestType = (values) => {
     this.update({
-      InterestType: values
+      InterestType: values,
     })
   }
 
-  updateAcquired(values) {
+  updateAcquired = (values) => {
     this.update({
-      Acquired: values
+      Acquired: values,
     })
   }
 
-  updateHowAcquired(values) {
+  updateHowAcquired = (values) => {
     this.update({
-      HowAcquired: values
+      HowAcquired: values,
     })
   }
 
-  updateCost(values) {
+  updateCost = (values) => {
     this.update({
-      Cost: values
+      Cost: values,
     })
   }
 
-  updateCostEstimated(values) {
+  updateCostEstimated = (values) => {
     this.update({
-      CostEstimated: values
+      CostEstimated: values,
     })
   }
 
-  updateValue(values) {
+  updateValue = (values) => {
     this.update({
-      Value: values
+      Value: values,
     })
   }
 
-  updateValueEstimated(values) {
+  updateValueEstimated = (values) => {
     this.update({
-      ValueEstimated: values
+      ValueEstimated: values,
     })
   }
 
-  updateRelinquished(values) {
+  updateRelinquished = (values) => {
     this.update({
-      Relinquished: values
+      Relinquished: values,
     })
   }
 
-  updateRelinquishedNotApplicable(values) {
+  updateRelinquishedNotApplicable = (values) => {
     this.update({
       RelinquishedNotApplicable: values,
-      Explanation: values.applicable ? this.props.Explanation : {}
+      Explanation: values.applicable ? this.props.Explanation : {},
```

Comment:
It looks like this also needs to set `Relinquished` (the date itself isn't clearing out)